src/Scrutor/Scrutor.csproj

```diff
@@ -3,7 +3,7 @@
     <Description>Register services using assembly scanning and a fluent API.</Description>
     <VersionPrefix>3.3.0</VersionPrefix>
     <Authors>Kristian Hellang</Authors>
-    <TargetFrameworks>net461;netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
We can get rid of .NET 5 as it's out of support in 3 months. Let's keep the LTS targets 👍🏻

Reply:
done